### PR TITLE
Features page

### LINKED ISSUE #1794: Create features page for federalist.18f.gov
#### User Story

As someone interested in Federalist, I'm curious what Federalist provides.

#### Background

We heard from users that they tend to assume Federalist is a CMS. They were also confused about the steps they must take in order to use Federalist. We believe that by creating a separate product page (linked to from the homepage navigation), we will be able to streamline the homepage, clarify what Federalist is and (from a high level) how to use it, and create a dedicated space to outline features and services without overwhelming the homepage.

This will involve moving some of the content on the homepage to this page.

#### Acceptance Criteria

- [x] We have created content for a product page that outlines what Federalist is, how to use it (at a high level), and what cool features it includes.
- [x] We have created a product page that is linked from both navigation bars on the homepage that reflects [Features - Desktop copy 2](https://gsa.invisionapp.com/share/YZKCIN2SKEP#/screens/302673508).

_After evaluating, edit this part:_

#### Implementation outline (if higher than "low" effort):

- [x] Clarify primary audience
- [x] Outline priorities for page
- [x] Develop content
- [x] Discuss potential visuals with viz designer
- [x] Build page
- [x] Publish content to page
- [x] Remove content from homepage that is duplicative of features page, eliminating content that less technical users consider to be "hard"
- [x] Change `Documentation` navigation title
- [x] Change `Get started` navigation title
- [x] Add useful links back into homepage page copy, linking to docs site and other valuable artifacts/sites/documentation. 
- [x] Corey or Will to review homepage changes
- [x] Will to ask Aaron about the bottom of the features page
- [x] Corey to do one more features page pass.
- [x] Will to ask Jen about bottom of page.
- [x] Will to make a fixing edit for bottom of page.

### PR DESCRIPTION
**Pull request by adborden**
Resolves #1794

This is based on @annahsebok 's branch https://github.com/18F/federalist/pull/1870

[Preview](https://adb-federalist-features.app.cloud.gov/features/). Note this is not a full Federalist instance, so some things might not work. Don't try to login :)

### CODE REVIEW

**Comment by adborden** [comment]
@jenniferthibault @coreycaitlin this is ready for a look.

**Comment by wslack** [comment]
Noting explicitly that Jen isn't obligated to review!

File: api/controllers/main.js [hunk 1]
```diff
@@ -70,6 +70,12 @@ module.exports = {
     return res.render('content/contact.njk', context);
   },
 
+  features(req, res) {
```
Comment (jmhooper) [inline api/controllers/main.js]: Could we grab the part from the examples function above that redirects for authed users? Looks like this part isn't available from the authenticated navbar.

File: views/features.njk [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+        <li>No hidden charges.</li>
+        <li>All the storage and bandwidth you need.</li>
+        <li>Unlimited additional sites for your team or office.</li>
+        <li>Two weeks to execute an average contract.</li>
+      </ul>
+    </section>
+  </div>
+  <div class="well">
+    <center>
+      <strong>Try a free test site to see if Federalist is right for you.</strong>
```
Comment (jenniferthibault) [inline views/features.njk]: This needs to be larger than regular body copy, and should keep the bold font weight. In mockups, I used the `h5`,  but that might not be semantically appropriate. What else might we be able to use?

File: views/features.njk [hunk 3]
```diff
@@ -0,0 +1,184 @@
...
+    </section>
+  </div>
+  <div class="usa-grid">
+    <hr class="hr-light">
+    <section class="info-block">
+      <div class="usa-width-one-whole">
+        <h2>Customize and publish your site with confidence.</h2>
+        <p>With Federalist, you can:</p>
+        <ul class="checklist copy">
+          <li>Monitor your content creation process</li>
```
Comment (jenniferthibault) [inline views/features.njk]: Is it possible to position the icons to match the layout below for better visual alignment? There's a bit of an optical illusion happening that makes it seem like the text is drooping below the icon baseline. 

screen shot 2018-07-23 at 6 08 19 pm

File: views/features.njk [hunk 4]
```diff
@@ -0,0 +1,184 @@
...
+         </div>
+        <div class="usa-grid-full">
+          <div class="usa-width-one-third">
+            <div class="feature-group vertical">
+              <div class="feature-image small">
+                <img src="/images/icons/icon-clockback-color.svg" />
+              </div>
+              <div class="feature-text">
+                <h4>Simple version control</h4>
+                <p>Use GitHub, a web-based hosting service with built-in version control, for real-time publishing and easy collaboration</p>
```
Comment (jenniferthibault) [inline views/features.njk]: Add closing period to match the rest

**Comment by jenniferthibault** [comment]
One more thing I missed — the `Features` item should be available from the mobile nav too, but is missing in this PR.

File: views/features.njk [hunk 5]
```diff
@@ -0,0 +1,184 @@
...
+        <h2>Try it for free</h2>
+        <p>Contact the Federalist team to kick off your next static site project.</p>
+        <button><img src="/images/icons/icon-envelope.svg" /> Get in touch</button>
+      </div>
+    </div>
+  </div>
+  <div class="usa-grid">
+    <section class="info-block">
+      <a id="page-body"></a>
+      <h2>Save time and money with built-in compliance and security.</h2>
```
[the remark this answers is not in the document]
Comment (coreycaitlin) [reply views/features.njk]: 👍Yep, agreed!

[the remark this answers is not in the document]
Comment (coreycaitlin) [reply views/features.njk]: ✅ done!

File: views/features.njk [hunk 6]
```diff
@@ -0,0 +1,184 @@
...
+          </div>
+        </div>
+      </div>
+    </section>
+  </div>
+  <div class="usa-grid">
+    <hr class="hr-light">
+    <section class="info-block">
+      <h2>It’s easy to get started.</h2>
+      <p>We’ve streamlined our onboarding process to get you working quickly:</p>
```
[the remark this answers is not in the document]
Comment (coreycaitlin) [reply views/features.njk]: ✅

**Comment by coreycaitlin** [comment]
I've made some edits and implemented a few of Jen's review comments — I'm going to step away now, but am happy to do a final pass if we can update the preview.

**Comment by adborden** [comment]
I pushed some more fixes and rebased. This is ready for another look @wslack @jmhooper

**Pull request merged**